### DISCUSSION TITLE
集录祈愿出了，UIGF v3.0 是不是也得跟进一下

### DISCUSSION THREAD

**Discussion by BTMuli** [UIGF-org/.github#14] [Ideas]
根据 Holo 佬的信息，新卡池代码是 `GACHA_TYPE_ACTIVITY_CITY 500`，而且根据 [官方说明](https://www.miyoushe.com/ys/article/49868038) ，混杂了角色跟武器，但是定轨后就是定角色歪角色、定武器歪武器

所以 @UIGF-org/uigf-constitutors 新标准应该还是比较容易适配的

**Comment by MoRanYue** [comment]
（好耶）

**Comment by BTMuli** [comment]
https://github.com/UIGF-org/UIGF-org.github.io/pull/52
